prime-router/pom.xml

```diff
@@ -166,7 +166,7 @@
                     <archive>
                         <manifest>
                             <addClasspath>true</addClasspath>
-                            <classpathPrefix>${stagingDirectory}/lib</classpathPrefix>
+                            <classpathPrefix>lib/</classpathPrefix>
```

Comment:
Not sure about this change. It breaks what I setup for the prime script. Why?

Reply:
thats a hold over from the older issues i was having with Windows...didnt realize that was in there...still